miscellaneous/miscellaneous.js

```diff
@@ -0,0 +1,150 @@
+{
+    let eightballPhrases = [
+        'As I see it, yes.',
+        'Ask again later.',
+        'Better not tell you now.',
+        'Cannot predict now.',
+        'Concentrate and ask again.',
+        'Don’t count on it.',
+        'It is certain.',
+        'It is decidedly so.',
+        'Most likely.',
+        'My reply is no.',
+        'My sources say no.',
+        'Outlook not so good.',
+        'Outlook good.',
+        'Reply hazy, try again.',
+        'Signs point to yes.',
+        'Very doubtful.',
+        'Without a doubt.',
+        'Yes.',
+        'Yes – definitely.',
+        'You may rely on it.'
+    ];
+
+    let RPS =[
+        'Rock',
+        'Paper',
+        'Scissors'
+    ];
+
+    let RPSScore = [];
+
+
+    /***
+     * Gets message from 8Ball
+     * returns Returns random string.
+     * */
+    function EightBall() {
+        return eightballPhrases[Math.floor(Math.random()*eightballPhrases.length)];
```

Comment:
could be reading this wrong, but this makes it so phrases can be easily added without breaking code, correct? cause it grabs the length of the phrases list then makes that the number that is the ceiling for the random function?

Reply:
correct, ideally I'd want this in a DB